client/fcm.js

```diff
@@ -0,0 +1,34 @@
+import firebase from "firebase/app";
+import "firebase/messaging";
+
+export async function initializeFCM(messageSenderId) {
+  if (!messageSenderId || isNaN(messageSenderId )){
```

Comment:
accept the messageSenderId as a string, since initializeApp expects it to be a string.